README.md

```diff
@@ -1,23 +1,11 @@
 # q1k_eeget_init
 Initialization code for the EEG/ET data from the Q1K project.
 
-# setup
-
-Assumes that https://github.com/jadesjardins/mne-python.git is a fork of git@github.com:scott-huberty/mne-python.git
-
 ## clone stuff
 ```bash
 git clone https://github.com/lina-usc/q1k_eeget_init.git
-cd q1k_eeget_init
-git clone https://github.com/jadesjardins/mne-python.git
-```
 
-## checkout the mne-eyetrack-revisions branch of scotts_fork
-```bash
-cd mne-python
-git remote add scotts_fork git@github.com:scott-huberty/mne-python.git
-git fetch scotts_fork
-git checkout --track scotts_fork/mne-eyetrack_revisions
```

Comment:
since mne-python/main has the eyetracking capability we dont need to use this fork anymore. removed.